### ISSUE TITLE
Model Project Feedback

### ISSUE THREAD

**Issue by Holger-Harmsen** [NumEconCopenhagen/projects-2020-green-mamba#9]
Cool parts:
- Good introduction, nice description of all equations and buildup of the model.
- Really nice usage of sympy, both solving, differentiating etc. 
- Nice using an interactive graph for the model
- Nice extension with human capital

Unclear parts:
I feel like the other team I gave feedback to made exactly the same outline as you, it is unclear if it’s an old exam project or you were inspired by the same, or just random. So I have some of the same feedback for you:
- It is cool that you extend the model - however, you do the exact same steps again; it would be cool with some different analysis the second time around;
- Could be nice with a simulation over time with stochastic shock maybe - or just showing convergence back to SS after a shock. It is a lot of work just for a transition diagram.

In conclusion:
- Nice project, great use of sympy and cool you’re creating interactive graphs. The project could be extended with time series or shocks.
